src/osd/OpRequest.h

```diff
@@ -63,6 +63,7 @@ struct OpRequest : public TrackedOp {
   entity_inst_t req_src_inst;
   uint8_t hit_flag_points;
   uint8_t latest_flag_point;
+  const char* last_event_detail;
```

Comment:
We can just initialize it  here -- it's less brittle if someone needs to add a constructor later.

last_event_detail = nullptr;